atd-vze/src/views/Crashes/Crash.js

```diff
@@ -278,7 +278,7 @@ function Crash(props) {
         <Col xs="12" md="6" className="mb-4">
           <CrashDiagram
             crashId={crashId}
-            isCr3Stored={cr3StoredFlag === "Y"}
+            isCr3Stored={cr3StoredFlag === "Y" && !!cr3FileMetadata}
```

Comment:
@johnclary I was reviewing this, and I wanted to double check that this is working like how we think. I checked out https://visionzero.austin.gov/editor/#/crashes/15208414 in prod. There is a pdf in S3 for it, but it matches `cr3_stored_flag = Y` and `cr3_file_metadata IS NULL`. I might be misunderstanding when we want to hide the button.

Reply:
thats an interesting point mike... thats also the crash that i checked out, i didnt realize we actually do have a cr3 for it. in this case it would be nice to be able to download that cr3 even though the metadata column is null?

Reply:
also, so far xavier letting us know when hes getting the `The specified key does not exist` error has helped flag underlying issues in our import script or cr3 downloader.. so maybe just hiding the button could also bite us and let things going wrong under the hood slip thru the cracks

Reply:
ugh, thank you. i got myself confused about the underlying issue w/ downloading CR3s from prod. the real problem is/was that the `cr3_stored_flag` was just plain wrong. the existing behavior is what we want. i'm closing this 😞